app/assets/stylesheets/application.scss

```diff
@@ -15,7 +15,54 @@
  *= require foundation_and_overrides
 
  */
+body {
+    background-color: rgb(211, 210, 210);
+}
```

Comment:
Your own styles should go in a separate file - this `application.scss` is a manifest file to help load in all of our CSS. To best organize our CSS, our styles should go in a new file within this `stylesheets` folder, like a `main.scss` file. You shouldn't need to include any import statements for the application to read the file, because that's one of the responsibilities of this file!

This manifest first `requires` foundation, then it `requires` all (`.`) files from this directory tree (i.e. the `stylesheets` folder), and lastly it `requires` this file. Adding styles here should be a last resort. Instead we can organize our styles into files. Right now is fine if all of your styles are in the same file, but you can also consider creating separate files that are meant for styling different components, that way we have a separation of concerns and it's easier to go back to find the styles we want to modify!